keyboards/omkbd/ergodash/rev1/rev1.c

```diff
@@ -8,7 +8,7 @@
 void matrix_init_kb(void) {
 
     #ifdef AUDIO_ENABLE
-        _delay_ms(20); // gets rid of tick
+        wait_ms(20); // gets rid of tick
```

Comment:
> At some point along the way, I made some minor adjustments to the ergodash firmware so that it builds with AUDIO_ENABLE = yes.

`make omkbd/ergodash/rev1:default AUDIO_ENABLE=yes` builds fine without these changes. Though if they are actually required for some other scenario i've missed, they should be submitted separately.

Reply:
🤷 . Ok; reverted.